spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -573,3 +573,69 @@ def expect_rate_limited_header(expected_to_be_present)
     end
   end
 end
+
+RSpec.describe 'when Socure throws an internal error' do
```

Comment:
I would not expect a second top-level describe in a file like this? Is there a way to nest that under the same existing one?

Reply:
Like if the scope/context/setup are so different in this block, maybe it should just be a new file?